### PR TITLE
Add subtitles for the what-is-monero video in home

### PR DESCRIPTION
**Pull request by erciccione**
Subtitles provided by @staff91

More subtitles are likely to arrive soon, i'll look into improving the videos. The subtitles format is `.vtt`, which is not supported by weblate, so at least for now translations must be submitted to this repo.

I'm having weird results on chromium, but that could be related to my local setup. So please test on your browsers. [There shouldn't be compatibility issues](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track).

### CODE REVIEW

**Comment by staff91** [comment]
Here's the translation to Italian for the video (.SRT format): https://gist.github.com/staff91/e64d6923dffb2b6162e23feca2d851af#file-what-is-monero-srt

**Comment by erciccione** [comment]
@staff91 Thanks, but please open a new issue about it. Also, the format of the subtitles must be `.vtt`

File: media/subtitles/en/monero-promo.vtt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+WEBVTT
+Kind: captions
+Language: en
+
+00:00:00.400 --> 00:00:06.320
+To most people, financial privacy is very important, but banks, big corporations and
+What Is Monero.vtt
```
Comment (erciccione) [inline media/subtitles/en/monero-promo.vtt:7]: This line has to be removed. Note to myself

**Comment by rehrar** [comment]
Plays fine for me in Firefox, Chrome, and Edge.

**Pull request merged**